brainstormforce/ultimate-addons-for-gutenberg
# Fix the issue of content not saving for content timeline.

### Description

* Fix the issue of content not saving for content timeline.
* https://trello.com/c/qykJcBQY/183-issue-of-content-not-saving-for-content-timeline
### Types of changes




* Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop content timeline.
- Edit Timeline heading and content save and check front-end.

### Checklist:
